spec/validators/status_prerequisite_validator_spec.rb

```diff
@@ -27,7 +27,56 @@
       expect(step_by_step_page.errors.messages[:status]).to eq([error_message])
     end
 
-    it "does not allow status to be in_review if the current status is not submitted_for_2i" do
+    it "does not allow status to be in_review if the current status is approved_2i" do
```

Comment:
There's a lot of scenarios here that are using the same code, but just with a different status value.

Could it be iterated over, like we've done with the user permission tests, e.g. https://github.com/alphagov/collections-publisher/blob/a167d3233451f326689c72d65a110f737f1e1b40/spec/controllers/review_controller_spec.rb#L98-L106